### PR TITLE
Add SPANK token

### PR DESCRIPTION
**Pull request by ameensol**

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1067,6 +1067,61 @@
...
+    {
+      "chainId": 10,
+      "address": "0x3bb4445d30ac020a84c1b5a8a2c6248ebc9779d0",
+      "name": "Theranos Coin",
+      "symbol": "LIZ",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/LIZ.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1080]: Copy-paste gone wrong? We already have Theranos Coin in the token list.

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -1067,6 +1067,61 @@
...
+    },
+    {
+      "chainId": 1,
+      "address": "0x42d6622dece394b54999fbd73d108123806f6a18",
+      "name": "SPANK",
+      "symbol": "SPANK",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/spank.png",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1088]: Logo should be closer to 256x256 size

File: optimism.tokenlist.json [hunk 3]
```diff
@@ -1067,6 +1067,61 @@
...
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
+    {
+      "chainId": 42,
+      "address": "0x2eA8f9B29AC1d70e01AE9D23d0451D2242e8d609",
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1106]: Verify this token in Etherscan

File: optimism.tokenlist.json [hunk 4]
```diff
@@ -1067,6 +1067,61 @@
...
+    },
+    {
+      "chainId": 69,
+      "address": "0xcfD1D50ce23C46D3Cf6407487B2F8934e96DC8f9",
+      "name": "SPANK",
+      "symbol": "SPANK",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/spank.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x22F24361D548e5FaAfb36d1437839f080363982B"
```
Comment (elenadimitrova) [inline optimism.tokenlist.json:1123]: The kovan bridge addresses are the wrong way around. On L2 - network 69 the `optimismBridgeAddress` is at `0x4200000000000000000000000000000000000010` and on L1 - network 42 it's at `0x22F24361D548e5FaAfb36d1437839f080363982B`

**Review by elenadimitrova — CHANGES_REQUESTED** [review]

File: optimism.tokenlist.json (hunk 1 above)
Comment (ameensol) [reply optimism.tokenlist.json:1080]: F, fixing

File: optimism.tokenlist.json (hunk 4 above)
Comment (ameensol) [reply optimism.tokenlist.json]: fixed

File: optimism.tokenlist.json (hunk 3 above)
Comment (ameensol) [reply optimism.tokenlist.json:1095]: fixed

File: optimism.tokenlist.json (hunk 2 above)
Comment (ameensol) [reply optimism.tokenlist.json:1077]: fixed

File: optimism.tokenlist.json (hunk 1 above)
Comment (ameensol) [reply optimism.tokenlist.json]: all done

File: optimism.tokenlist.json (hunk 3 above)
Comment (elenadimitrova) [reply optimism.tokenlist.json:1095]: Can you give my account 0xe3a90926133fef3c58514fA1aAeF75E0998aedD5 some kovan tokens I can test with? This is all in the repo readme

**Comment by elenadimitrova** [comment]
@ameensol Can you give my account 0xe3a90926133fef3c58514fA1aAeF75E0998aedD5 some kovan tokens I can test with?

**Comment by ameensol** [comment]
did the rebase work or do you want me to get rid of the commit history as well?

**Comment by elenadimitrova** [comment]
@ameensol the rebase worked but you still have not given my account kovan tokens to test with as I requested a week ago here https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/66#discussion_r785756491 and then a few days back again here https://github.com/ethereum-optimism/ethereum-optimism.github.io/pull/66#issuecomment-1015346302

**Comment by ameensol** [comment]
Hm I sent some tokens to your address on kovan: https://kovan.etherscan.io/tx/0xe5477708ac92eacf270199ba75bac3f7c5059643240845c20d419b0dd680f828

Did you need them on the kovan optimism?

**Review by elenadimitrova — APPROVED** [review]

**Comment by elenadimitrova** [comment]
@ameensol got the tokens thanks. Approved the PR, please rebase and I'll merge

**Pull request closed without merging**